src/ViewModels/SchemaTypesViewModel.cs

```diff
@@ -0,0 +1,279 @@
+                                {
+                                    string returnTypeFullName = operation.ReturnType.ToStructuredType()?.FullTypeName() ?? operation.ReturnType.FullName();
+                                    AddRelatedType(returnTypeFullName, structuredType.FullTypeName());
+
+                                    if (SchemaTypeModel.TryGetValue(returnTypeFullName, out SchemaTypeModel referencedschemaTypeModel)
+                                        && !referencedschemaTypeModel.IsSelected)
+                                    {
+                                        referencedschemaTypeModel.IsSelected = true;
+                                    }
+                                }
+
+                                IEnumerable<IEdmOperationParameter> parameters = operation.Parameters;
+
+                                foreach (var parameter in parameters)
+                                {
+                                    if (parameter.Type.ToStructuredType() != null || parameter.Type.IsEnum())
+                                    {
+                                        string parameterFullName = parameter.Type.ToStructuredType()?.FullTypeName() ?? parameter.Type.FullName();
+                                        AddRelatedType(parameterFullName, structuredType.FullTypeName());
+
+                                        if (SchemaTypeModel.TryGetValue(parameterFullName, out SchemaTypeModel model) && !model.IsSelected)
+                                        {
+                                            model.IsSelected = true;
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    }
+                };
+
+                toLoad.Add(schemaTypeModel);
+                schemaTypeModel.IsSelected = true;
+                SchemaTypeModel.Add(schemaType.FullTypeName(), schemaTypeModel);
+            }
+
+            SchemaTypes = toLoad.OrderBy(o => o.Name).ToList();
+        }
+
+        public void AddRelatedType(string type, string relatedType)
+        {
+            if (RelatedTypes.TryGetValue(type, out ICollection<string> relatedTypes))
+            {
+                relatedTypes.Add(relatedType);
+            }
+            else
+            {
+                relatedTypes = new HashSet<string>();
+                relatedTypes.Add(relatedType);
+                RelatedTypes.Add(type, relatedTypes);
+            }
+        }
+
+        public void ExcludeSchemaTypes(IEnumerable<string> schemaTypesToExclude)
+        {
+            foreach (var schemaTypeModel in SchemaTypes)
+            {
+                schemaTypeModel.IsSelected = !schemaTypesToExclude.Contains(schemaTypeModel.Name);
+            }
+        }
+
+        public void EmptyList()
```

Comment:
I
```suggestion
        public void ClearSchemaTypes()
```
EmptyList is kind of ambiguous

Reply:
Fixed.